sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -29,9 +29,10 @@
  * for that.
  */
 extern manifest_t _manifest;
-extern uint32_t _vflash_start;
-extern uint32_t _vflash_size;
-extern uint32_t _flash_start;
+// TODO(lowrisc/opentitan:#10712): setup Ibex address translation
```

Comment:
nit: I recommend to reference a previous commit in the issue to reference the vflash patch rather than committing commented out code.

Reply:
Ah SG, thanks. Fixed.